### PR TITLE
propose edits for tiny typographical/grammatical errors

### PR DESCRIPTION
**Pull request by zipperer**

### CODE REVIEW

File: chapter_03_unit_test_first_view.asciidoc [hunk 1]
```diff
@@ -14,7 +14,7 @@ We finished the last chapter with a functional test failing,
 telling us that it wanted the home page for our site to have ``To-Do'' in its title.
 It's time to start working on our application.
 In this chapter, we'll build our first HTML page, find out about URL handling,
-and creating responses to HTTP requests with Django's view functions.
+and create responses to HTTP requests with Django's view functions.
```
Comment (hjwp) [inline chapter_03_unit_test_first_view.asciidoc:17]: i think this one works either way - my way was "we'll find out about (a) url handling and (b) creating responses"

**Comment by hjwp** [comment]
thanks very much indeed!  how are you enjoying the book?

**Comment by zipperer** [comment]
I am enjoying it very much :)

Thank you for writing it and maintaining it.

### General
The book has been easy to read and follow. It points to documentation, blogs, and essays -- each of which encourage the reader to explore further. And its conversational tone enlivens the prose.

### Setting up development environment
I appreciate the care the book takes to help the reader set up an isolated virtual environment with python, django, git, and selenium -- each with its appropriate version.

### Hosting content from the book
Hosting the book at obeythetestinggoat.com/book/CHAPTER with the Disqus forum for each chapter provides a convenient way for readers to share notes with each other and you. Hosting the source for the book on GitHub (a) shows readers a method to maintain documents over the web and (b) enables readers to propose edits and apply git skills the book conveys. And hosting versions of the repository against which a reader can diff his own (as described in [this appendix](https://www.obeythetestinggoat.com/book/appendix_github_links.html#_using_git_to_check_your_progress)) is handy.

### Tests+commits serve as tools for focus+review
I am barely a novice in developing for the web and using web frameworks. So, I'm easily confused, and I'll need to review. The book uses tests and test failures to illustrate (a) the effects of each code change and (b) changes required to achieve a particular effect. This use of tests makes concrete the goal at each step in the book, and it leaves artifacts (tests and code changes bundled together in a git commit with a message) that the reader can review to remember how to add/edit behaviors.

### Onward
I've read only 6 chapters. So my compliments extend only to that extent. When I've read 20 more chapters and a few appendices, I'll return and give the book a thorough lashing ;)

File: chapter_03_unit_test_first_view.asciidoc (hunk 1 above)
Comment (zipperer) [reply chapter_03_unit_test_first_view.asciidoc:17]: I agree that both ways work well. For that way I would propose
```
In this chapter, we'll build our first HTML page and find out about (a) URL handling and (b) creating responses...
```
i.e. remove the comma between 'page' and 'find'.

**Comment by hjwp** [comment]
lol thanks very much - really appreciate you taking the time to write up your thoughts.  and the typo fixes too!  let me know how you enjoy the next few chapters for sure!

**Pull request merged**